lib/locale.ts

```diff
@@ -35,3 +35,19 @@ export const labels = {
   "zh-CN": "Chinese (Simplified)",
   "zh-TW": "Chinese (Traditional)",
 };
+
+export function createLocaleMap(locales: Array<keyof typeof labels>) {
+  const map: Record<string, string> = {};
+
+  for (const locale of locales) {
+    map[locale] = labels[locale] ?? locale;
+  }
+
+  return map;
+}
+
+export function getSortedLabels(map: Record<keyof typeof labels, string>) {
+  return Object.entries(map).sort(([_0, a], [_1, b]) =>
```

Comment:
Since this comparator function is specific to labels, I would change `a` and `b` to something more descriptive. I assume they're the names of labels? Maybe something like `labelNameA`?